File: lecture0-30/lecture011/display.java
import java.util.Scanner;
public class display{
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args){
        int [] arr = {1,2,3,4,5,6,7,89};
        display(arr , 0); 
    }
    public static void display(int [] arr , int idx){
        if(idx==arr.length)
        return ;

        System.out.print(arr[idx] + " ");
        display(arr , idx+1);

    } 
}

File: lecture0-30/lecture011/display.cpp
#include<iostream>
#include<vector>
using namespace std ;
void display(vector<int> arr , int idx){
    if(idx==arr.size())
    return ;

    cout<<arr[idx]<<" ";
    display(arr , idx+1);
}
int main(int args , char** argv){
    vector<int> arr={10,20,30,40,50,60,70,80};
    display(arr , 0);   
}